docs/UserGuide.md

```diff
@@ -154,19 +154,19 @@ Here are the common terms that is used throughout this user guide.
 
 #### Command format
 
-* Words in `UPPER_CASE` are compulsory parameters to be supplied by the user.<br>
-  E.g. in `add /c CLASS_NAME` command, you must provide the `CLASS_NAME`.
+* Words in `UPPER_CASE` are compulsory parameters to be supplied.<br>
+  E.g., you are to provide the `CLASS_NAME` in `add /c CLASS_NAME` command.
 
 * Words in `UPPER_CASE` enclosed in square brackets are optional parameters. <br>
-  E.g: in `edit /c CLASS_INDEX [/n CLASS_NAME] [/m MEMO] [/t SCHEDULE]` command, `CLASS_NAME`, `MEMO`, `SCHEDULE` are optional parameters.
+  E.g., you can choose to provide `CLASS_NAME`, `MEMO` and `SCHEDULE` in `edit /c CLASS_INDEX [/n CLASS_NAME] [/m MEMO] [/t SCHEDULE]` command
 
 * Optional parameters can be in any order.<br>
-  E.g. if the command specifies `[/n CLASS_NAME] [/m MEMO]`, `[/m MEMO] [/n CLASS_NAME]` is also acceptable.
+  E.g., you can choose to provide in the following order:`[/m MEMO] [/n CLASS_NAME]` if the command specifies `[/n CLASS_NAME] [/m MEMO]`.
```

Comment:
i think it would be a bit clearer if you rearrange the sentence like this, you can consider whether to fix or not:
"if the command specifies `[/n CLASS_NAME] [/m MEMO]` , you can choose to provide in the following order:`[/m MEMO] [/n CLASS_NAME]`"

Reply:
Okay, will change. I was trying to make it fit the previous 2 example messages but I think I am pushing it too hard LOL